src/main/scala/cognite/spark/v1/SequenceRowsRelation.scala

```diff
@@ -228,16 +228,9 @@ class SequenceRowsRelation(config: RelationConfig, sequenceId: CogniteId)(val sq
         .toList
         .parTraverse {
           case (cogniteExternalId, rows) =>
```

Comment:
Looks like this is incorrectly named and should be `cogniteId` (or just `id`) instead.

Reply:
correct, updated :)